webpack/assets/javascripts/react_app/components/Pagination/Pagination.js

```diff
@@ -17,6 +17,8 @@ const Pagination = props => {
     onPageSet,
     onPerPageSelect,
     dropdownButtonId,
+    disableNext,
```

Comment:
Is this apart of the scope? Can it be done in a different PR?

Reply:
its part of the scope yes, adding foundation for router pagination.